internal/backend/s3/config_test.go

```diff
@@ -111,3 +114,14 @@ func TestParseConfig(t *testing.T) {
 		}
 	}
 }
+
+func TestParseError(t *testing.T) {
```

Comment:
That test case never enters `createConfig` only `ParseConfig` where it fails at the check which looks for a `s3:` prefix. Judging from the test patterns, it looks like the intention was to reach `createConfig`?